### ISSUE TITLE
Feature: Add options to compress screenshot to jpeg

### ISSUE THREAD

**Issue by rawpixel-vincent** [tacoss/testcafe-blink-diff#53]
Currently the screenshot are all saved in png format without compression.

Add options to compress and save the screenshots jpeg.

replace png libraries with sharp to get jpg and png support and optimize performance.

@pateketrueke if you are good with this, you can assign it to me, will work on that together with resizing targets for diff #48

**Comment by pateketrueke** [comment]
Agreed, sorry for the late response, go ahead as it sound it'll rocks!

**Comment by rawpixel-vincent** [comment]
cool, I will start on this once the current PR are ok and merged in dev branch
